lib/ClimaLandSimulations/experiments/ozark.jl

```diff
@@ -1,6 +1,21 @@
 using ClimaLandSimulations
+using ClimaLandSimulations.Fluxnet
 
-sv, sol, Y, p, inputs, climaland = fluxnet_simulation("US-MOz");
+# default parameters, except for custom parameter hetero_resp.b
+sv_test, sol_test, Y_test, p_test = fluxnet_simulation(
+    "US-MOz";
+    params = ozark_default_params(; hetero_resp = hetero_resp_ozark(; b = 2)),
```

Comment:
should this be the default? if this is testing that we can add in other parameters, maybe that should go into a unit test. [later PR ok]

Reply:
what do you mean by should this be the default? 

I agree this should eventually become a unit test - maybe later PR